lib/PanelTestProvider.jsm

```diff
@@ -60,6 +61,21 @@ const MESSAGES = () => [
     targeting: `!hasAccessedFxAPanel && !usesFirefoxSync && isFxAEnabled == true`,
     trigger: { id: "toolbarBadgeUpdate" },
   },
+  {
+    id: "WNP_PROFILE_AGE",
+    template: "update_action",
+    content: {
+      action: {
+        id: "moments-wnp",
+        data: {
+          url:
+            "https://www.mozilla.org/en-US/firefox/80.0a1/whatsnew/?oldversion=50.0a2",
```

Comment:
You could go with `https://www.mozilla.org/en-US/etc/%LOCALE%/retention/thank-you-a/` for something a little different looking